meerk40t/core/elements.py

```diff
@@ -5406,68 +5422,145 @@ def clear_all_regmarks(node, **kwargs):
         # Calculate the amount of selected nodes in the tree:
         # If there are ops selected then they take precedence
         # and will only be counted
+
         @self.tree_conditional(
             lambda cond: len(
-                list(self.flat(selected=True, cascade=False, types=operate_nodes))
+                list(self.flat(selected=True, cascade=False, types=("reference")))
```

Comment:
`("reference")` is still just `"reference"` which is notably different than `("reference",)` though the code there accepts a singe string without issue and does require a tuple for types.